cms/static/js/views/course_info_update.js

```diff
@@ -17,32 +23,53 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
 
             initialize: function() {
                 this.template = this.loadTemplate('course_info_update');
-                this.render();
-            // when the client refetches the updates as a whole, re-render them
+                // Disable the new update button while we load the collection.
+                this.$el.find('.new-update-button').prop('disabled', true);
```

Comment:
How are you specifying disabled in the HTML? See http://www.w3schools.com/tags/att_disabled.asp for the correct way of specifying it.